ouroboros-network/test/Test/Ouroboros/Network/PeerSelection/RootPeersDNS.hs

```diff
@@ -676,7 +688,7 @@ prop_local_singleSourceOfTruth mockRoots
       $ foldl'
           (\(r, prev) t ->
             case t of
-              TraceLocalRootGroups curr ->
+              Solo curr ->
```

Comment:
This should be `Solo (TraceLocalRootGroups curr)`, isn't it?

Reply:
No, I am tracking it like `Solo curr`.

Reply:
Yes, it makes more sense to do so.